programs/survival/cam.sc

```diff
@@ -59,6 +59,8 @@ __command() ->
             display_title(p, 'actionbar', format('y Entered camera mode'));
          )
       ));
+   , current_gamemode == 'creative',
```

Comment:
At this point you might as well not add this, and just leave it as an end 'else' case, since even in adventure mode won't want to allow to use /cam